views.py

```diff
@@ -401,8 +401,8 @@ def export_articles_all(request):
     )
 
     # Handle stage without elements
-    if stage in ['Published', 'Rejected']:
-        articles = articles.filter(stage=element)
+    if stage in ['Production', 'Proofing', 'Published', 'Rejected']:
```

Comment:
production and proofing do have elements, why do we need to add them to this list?